packages/fuels-signers/src/wallet.rs

```diff
@@ -109,8 +100,14 @@ impl From<WalletError> for Error {
 }
 
 impl Wallet {
-    /// Create a new wallet from a private key.
-    pub fn new_from_private_key(private_key: SecretKey, provider: Provider) -> Self {
+    pub fn new_random(provider: Option<Provider>) -> Self {
+        let mut rng = rand::thread_rng();
+        let private_key = SecretKey::random(&mut rng);
+
+        Self::new_from_private_key(private_key, provider)
+    }
+
+    pub fn new_from_private_key(private_key: SecretKey, provider: Option<Provider>) -> Self {
```

Comment:
I like this; if you want a quick wallet without thinking too much: `new_random(None)`, otherwise, you have many other ways of constructing a wallet: PKs, mnemonic, encrypted JSON wallet. 👍